pkgs/tools/misc/ksnip/default.nix

```diff
@@ -0,0 +1,77 @@
+{ stdenv
+, lib
+, cmake
+, extra-cmake-modules
+, fetchFromGitHub
+, kColorPicker
+, kImageAnnotator
+, qtsvg
+, qttranslations
+, qtx11extras
+}:
+
+stdenv.mkDerivation rec {
+  pname = "ksnip";
+  version = "1.9.1";
+
+  src = fetchFromGitHub {
+    owner = "ksnip";
+    repo = "ksnip";
+    rev = "v${version}";
+    sha256 = "1izsk586n9fbm0di0hj6pxs7r0a6w554gpad1ghf247icr0pfc1l";
+  };
+
+  dontWrapQtApps = true;
+
+  nativeBuildInputs = [
+    cmake
+    extra-cmake-modules
+  ];
+
+  buildInputs = [
+    kColorPicker
+    kImageAnnotator
```

Comment:
I just stumbled upon an existing `kcolorpicker` and `kimageannotator` in `pkgs/top-level/qt5-packages.nix`. I will switch to those if possible.